src/main/java/Duke/DeadLine.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+/**
+ * The DeadLine class extends the Task Class. The DeadLine class is associated with Tasks that have a deadline.
+ * Hence, the LocalDateTime attribute of this class is derived from what comes after "/by" in the user's command
+ * <p>The Event class contains a LocalDateTime object</p>
+ */
+
+public class DeadLine extends Task {
```

Comment:
Deadline is one single word to the class should be named `Deadline` with a small `l`.